.github/workflows/docs.yml

```diff
@@ -9,11 +9,32 @@ on:
       publish:
         default: false
         type: boolean
+      version:
+        default: ${{ github.ref_name }}
```

Comment:
Maybe we can't use ``github.ref_name`` here...?
The CI failed complaining about ``github`` in this line.